Documentation/Signoffs/detaildesgincamera.md

```diff
@@ -0,0 +1,137 @@
+<br />
+
+2. Monitoring
+    * The measurements for a single parking spot in the Bell Hall lot is 575 cm x 265 cm. Based on these measurements the whole area of the lot is then calculated in Figure 3.  The computer vision algorithms implemented [1] and [2] which detect and/or identify vehicles used training set images of the rear side of cars that were 64 x 64 pixels. A standard car is on average 5.8 feet [3], which translates to roughly a little over 10 pixels per foot (or 100 pixels per square foot) for these images of cars to be identified as such. Thus, the cameras must have a minimum pixels per foot of 10 pixels per foot (or 100 pixels per/square foot) to identify the presence of a car in a parking spot. Using 100 pixels/sq ft as a bare minimum for vehicle detection and mapping the area of the parking lot to be 23,261 sq ft. A total of 2.33 megapixels will be necessary for the whole lot. This includes the driving area, grass/shrubbery, and curbs around the area. Then, using the arrangement above, a total of nine cameras will be used. This suggests that each camera has a bare minimum resolution of 0.26 megapixels. The camera that we have chosen is 3 megapixels, which is above the required minimum.
+
+
+
+![Figure 2. Camera Arrangement](../Images/Bell_Hall_Lot_Sectors.png)
+<h3>Figure 2. Camera Arrangement</h3>
+
+![Figure 3. Bell Hall Lot Area](../Images/Bell_Hall_Lot_Area.png)
+<h3>Figure 3. Bell Hall Lot Area</h3>
+
+<br />
+
+3. Communication
+    * The camera that we have chosen has 2.4G WiFi/cloud capabilities, which will allow for wireless communication without any external devices.
+4. Field of View
+    * The camera that we have chosen has a 130 degree field of view and can pan up to 360 degrees horizontally and 120 degrees vertically. These features may not be utilized, but the camera was chosen due to the 130 degree field of view. The maximum angle needed in any one of the individual sectors is 87.96 degrees, so by choosing a camera that is able to observe much more area than that, we can accurately get readings of the desired regions. In addition, the furthest point that a single camera may have to monitor is roughly 80-100 feet, which would be within a typical recognition range for the camera[4],  and placing the camera as high as 20 feet would provide a high view to see cars at this distance, especially if (in worst case) a smaller car (like a Nissan Sentra) were next to a taller truck (like a Toyota Tacoma).
+5. Maintainability
+    * This camera was chosen due to it having “3-8 months of standby time only on battery power, plus the solar panel can quickly achieve 365-day non-stop protection”. This allows for minimal maintenance in respect to power. The camera is also IP66 waterproof (tested in high-pressure water conditions. 
+6. Power
+    * The camera has its own power system between the battery which can solely last 3-8 months without being recharged. Then, with the solar panel, it will be able to run year round. 
```

Comment:
No schematic for adding external power. I'm not saying you have to design the solar system. However, you have not shown a schematic for how connection to the camera's power are possible.